mne/decoding/tests/test_search_light.py

```diff
@@ -113,6 +120,13 @@ def transform(self, X):
         assert_array_equal(features_shape, [3, 4])
     assert_array_equal(y_preds[0], y_preds[1])
 
+    # Test when 'roc_auc' for binary labels
+    X = np.random.rand(5, 20, 2)
+    y = [4, 4, 3, 3, 3]
+    sl = _SearchLight(LogisticRegression(), scoring='roc_auc')
+    sl.fit(X, y)
+    sl.score(X, y)
```

Comment:
I think there's still a problem. If you fit with y in [4, 3], and score (without an roc_auc), score will change y_true in [4, 3] into y_true in [0, 1], but the estimator predictions will in [4, 3] won't they?

You need to add a test where X is not fully random but has some information, and check that roc_auc or accuracy give higher than chance scores.